src/filters/resources.js

```diff
@@ -133,4 +133,15 @@ angular.module('openshiftCommonUI')
   })
   .filter('isBindingReady', function(isServiceInstanceReadyFilter) {
     return isServiceInstanceReadyFilter;
-  });
+  })
+  .filter('hasDeployment', function(annotationFilter) {
+    return function(object) {
+      return !!annotationFilter(object, 'deployment.kubernetes.io/revision');
+    };
+  })
+  .filter('hasDeploymentConfig', function(annotationFilter) {
+    return function(deployment) {
+      return (annotationFilter(deployment, 'deploymentConfig')) ? true : false;
```

Comment:
I realize this is copied code, but I suggest we either use `!!` or ternary for both filters. I have a slight preference for `!!` but I'm OK either way :)